docs/workforce-management-framework/topics/con-overview.adoc

```diff
@@ -0,0 +1,21 @@
+[id='Overview-{chapter}']
+= Overview
+
+This section provides a overview of {WFM-RC-NameLong} documentation.
+You can read this document in a linear fashion, or you can skip sections if something doesn’t interest you.
+
+==  About the documentation
+
+The  {WFM-RC-NameLong} documentation is available as html files generated from asciidocs documents.
+You can view source code of this page on github: https://feedhenry-raincatcher.github.io/raincatcher-docs/
+
+== Getting started
+
+To quickly start write mobile enabled Workflow Management applications please follow our guides:
+
+- link:./guides/wfm.html[WFM Guide]
+Getting Started for Workflow Management Solutions.
```

Comment:
Do we still support this? I think that actually modular documentation it's one user oriented guide :)